pom.xml

```diff
@@ -467,7 +467,7 @@
           <dependency>
             <groupId>com.puppycrawl.tools</groupId>
             <artifactId>checkstyle</artifactId>
-            <version>7.8</version>
+            <version>7.9-SNAPSHOT</version>
```

Comment:
> Maven release plugin is failing if any snapshot version is found

@Luolc So we can't keep this change. We will get around limitation by using #4403 then.
Passing CI is still a good sign that the fix will work.